spec/controllers/organisers_controller_spec.rb

```diff
@@ -0,0 +1,116 @@
+require 'rails_helper'
+
+RSpec.describe OrganisersController, type: :controller do
+	describe 'Organisers Controller GET Tests' do
+		before(:each) do
+			@organiser = create(:user)
+		end
+
+		it 'should get the organiser dashboard' do
+			edition = create(:edition)
+			login_user(@organiser)
+			get :dashboard, user: @organiser
+			expect(response).to have_http_status(200)
+		end
+
+		it 'should get organiser dashboard with "final_project_evaluation"' do
+			edition = Edition.create(name: "test",
+				start_date: 100.days.ago,
+				end_date: (10.days.ago),
+				registration_started_at: 200.days.ago,
+				registration_closed_at: 185.days.ago)
+			login_user(@organiser)
+			get :dashboard, user: @organiser
+			expect(response).to have_http_status(200)
```

Comment:
same here

Reply:
Added: 
`response.should render_template('organisers/final_project_evaluation')`